src/main/scala/tech/cryptonomic/conseil/routes/openapi/DataJsonSchemas.scala

```diff
@@ -1,51 +1,57 @@
 package tech.cryptonomic.conseil.routes.openapi
 
-import endpoints.{algebra, generic}
+import endpoints.generic
 import tech.cryptonomic.conseil.generic.chain.DataTypes._
 import tech.cryptonomic.conseil.tezos.FeeOperations.AverageFees
+import tech.cryptonomic.conseil.tezos.ApiOperations.{BlockResult, OperationGroupResult, AccountResult}
 import tech.cryptonomic.conseil.tezos.Tables.{AccountsRow, BlocksRow, OperationGroupsRow, OperationsRow}
 
 /** Trait containing Data endpoints JSON schemas */
-trait DataJsonSchemas extends algebra.JsonSchemas with generic.JsonSchemas {
+trait DataJsonSchemas extends generic.JsonSchemas {
 
   /** API query schema */
-  implicit def queryRequestSchema: JsonSchema[ApiQuery] =
+  implicit lazy val queryRequestSchema: JsonSchema[ApiQuery] =
     genericJsonSchema[ApiQuery]
 
   /** Query predicate schema */
-  implicit def queryPredicateSchema: JsonSchema[Predicate] =
+  implicit lazy val queryPredicateSchema: JsonSchema[Predicate] =
     genericJsonSchema[Predicate]
 
   /** Query ordering operation schema */
-  implicit def queryOrderingOperationSchema: JsonSchema[OperationType.Value] =
+  implicit lazy val queryOrderingOperationSchema: JsonSchema[OperationType.Value] =
     enumeration(OperationType.values.toSeq)(_.toString)
 
   /** Query ordering schema */
-  implicit def queryOrderingSchema: JsonSchema[QueryOrdering] =
+  implicit lazy val queryOrderingSchema: JsonSchema[QueryOrdering] =
     genericJsonSchema[QueryOrdering]
 
   /** Query ordering direction schema */
-  implicit def queryOrderingDirectionSchema: JsonSchema[OrderDirection.Value] =
+  implicit lazy val queryOrderingDirectionSchema: JsonSchema[OrderDirection.Value] =
     enumeration(OrderDirection.values.toSeq)(_.toString)
 
   /** Query output schema */
-  implicit def queryOutputSchema: JsonSchema[OutputType.Value] =
+  implicit lazy val queryOutputSchema: JsonSchema[OutputType.Value] =
     enumeration(OutputType.values.toSeq)(_.toString)
 
-
   /** Timestamp schema */
-  implicit def timestampSchema: JsonSchema[java.sql.Timestamp]
+  implicit lazy val timestampSchema: JsonSchema[java.sql.Timestamp] = {
+    xmapJsonSchema[Long, java.sql.Timestamp](
+      implicitly[JsonSchema[Long]],
+      millisFromEpoch => new java.sql.Timestamp(millisFromEpoch),
+      ts => ts.getTime
+    )
+  }
 
   /** Blocks row schema */
-  implicit def blocksRowSchema: JsonSchema[BlocksRow] =
+  implicit lazy val blocksRowSchema: JsonSchema[BlocksRow] =
```

Comment:
what was the impetus behind making these values lazy?

Reply:
They've always been (they where methods).
The idea was to compute them only once, as I expect the generically derived implementation might carry some extra weight.
I made them lazy to keep the lazy semantic nonetheless